build.gradle

```diff
@@ -21,16 +21,23 @@ buildscript {
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
         classpath 'gradle.plugin.org.spongepowered:spongegradle:0.8.1'
         classpath 'org.spongepowered:mixingradle:0.4-SNAPSHOT'
+        classpath mcTesterDep
     }
 }
 
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.2.51'
```

Comment:
Will this allow kotlin code in main source set, too? If yes we might want to avoid that.

Reply:
I don't believe so - all of the actual Kotlin library dependencies (e.g. `kotlin-stdlib`) are set to `testCompile`.